core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -505,6 +507,33 @@ var ExplorationEditorMainTab = function() {
       'State content editor takes too long to disappear');
   };
 
+  this.setContent1 = async function(richTextInstructions) {
```

Comment:
I don't think `setContent1` is a great name for this function, can we change it to something more descriptive?

Reply:
Done. Changed to `setContentForExplorationEditor`